go/cmd/vtctldclient/internal/command/vschemas.go

```diff
@@ -150,6 +179,12 @@ func init() {
 	ApplyVSchema.Flags().BoolVar(&applyVSchemaOptions.SkipRebuild, "skip-rebuild", false, "If set, do no rebuild the SrvSchema objects.")
 	ApplyVSchema.Flags().StringSliceVar(&applyVSchemaOptions.Cells, "cells", nil, "If specified, limits the rebuild to the cells, after upload. Ignored if skipRebuild is set.")
 	Root.AddCommand(ApplyVSchema)
-
+	// GetVSchema
 	Root.AddCommand(GetVSchema)
+	// ValidateVSchema
+	ValidateVSchema.Flags().StringVar(&validateVSchemaOptions.Keyspace, "keyspace", "", "Keyspace")
+	ValidateVSchema.Flags().StringSliceVar(&validateVSchemaOptions.Shards, "shards", []string{}, "runs validateVSchema on these specific shards")
```

Comment:
nil default

Reply:
also this help message refers to a private function, which isn't great from a UX perspective. i think we should just say "validate the vschema on the following shards"